### PR TITLE
refact: rename linuxNew to linux

### PR DESCRIPTION
**Pull request by zdtsw**

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM - but will hold off merging until downstream partners can prepare.

**Comment by karianna** [comment]
OK, MSFT have been warned, we're OK to merge this if IBM/Red Hat and others are.

**Comment by zdtsw** [comment]
i will post it on slack and wait till next Tue, since most are off the rest of the week.

**Review by gdams — APPROVED** [review]

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ name: Check Linux Packages
 on:
   push:
```
Comment (gdams) [inline .github/workflows/linux.yml:3]: while you're here would you mind removing the whole `push` block?

It seems necessary to run the build/test again especially as nobody is monitoring it's status once it's been merged

**Comment by karianna** [comment]
@zdtsw Will need a rebase.

File: .github/workflows/linux.yml (hunk 1 above)
Comment (zdtsw) [reply .github/workflows/linux.yml]: done

**Comment by zdtsw** [comment]
if everything is fine, can we have this one merged to master today? thanks!

**Pull request merged**